### ISSUE TITLE
Installation fails `Errno::ENOENT: No such file or directory - bin/ameba`

### ISSUE THREAD

**Issue by fnordfish** [crystal-ameba/homebrew-ameba#1]
The formula tries to copy a `bin/ameba` which seems not to be present in the tarball.

Trace:

```
/opt/homebrew/Library/Homebrew/ignorable.rb:27:in `block in raise'
/opt/homebrew/Library/Homebrew/ignorable.rb:26:in `callcc'
/opt/homebrew/Library/Homebrew/ignorable.rb:26:in `raise'
/opt/homebrew/Library/Homebrew/extend/pathname.rb:121:in `install_p'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/call_validation.rb:270:in `bind_call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/call_validation.rb:270:in `validate_call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
/opt/homebrew/Library/Homebrew/extend/pathname.rb:113:in `block in install'
/opt/homebrew/Library/Homebrew/extend/pathname.rb:94:in `each'
/opt/homebrew/Library/Homebrew/extend/pathname.rb:94:in `install'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/call_validation.rb:270:in `bind_call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/call_validation.rb:270:in `validate_call'
/opt/homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11600/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
/opt/homebrew/Library/Taps/crystal-ameba/homebrew-ameba/Formula/ameba.rb:21:in `install'
/opt/homebrew/Library/Homebrew/debrew.rb:15:in `block in install'
/opt/homebrew/Library/Homebrew/debrew.rb:86:in `debrew'
/opt/homebrew/Library/Homebrew/debrew.rb:15:in `install'
/opt/homebrew/Library/Homebrew/build.rb:179:in `block (3 levels) in install'
/opt/homebrew/Library/Homebrew/extend/kernel.rb:536:in `with_env'
/opt/homebrew/Library/Homebrew/build.rb:139:in `block (2 levels) in install'
/opt/homebrew/Library/Homebrew/formula.rb:1589:in `block in brew'
/opt/homebrew/Library/Homebrew/formula.rb:3246:in `block (2 levels) in stage'
/opt/homebrew/Library/Homebrew/extend/kernel.rb:536:in `with_env'
/opt/homebrew/Library/Homebrew/formula.rb:3245:in `block in stage'
/opt/homebrew/Library/Homebrew/resource.rb:128:in `block (2 levels) in unpack'
/opt/homebrew/Library/Homebrew/download_strategy.rb:116:in `chdir'
/opt/homebrew/Library/Homebrew/download_strategy.rb:116:in `chdir'
/opt/homebrew/Library/Homebrew/download_strategy.rb:103:in `stage'
/opt/homebrew/Library/Homebrew/resource.rb:124:in `block in unpack'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `block in run'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `chdir'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `run'
/opt/homebrew/Library/Homebrew/resource.rb:238:in `stage_resource'
/opt/homebrew/Library/Homebrew/resource.rb:123:in `unpack'
/opt/homebrew/Library/Homebrew/resource.rb:97:in `stage'
/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.5/lib/ruby/3.3.0/forwardable.rb:240:in `stage'
/opt/homebrew/Library/Homebrew/formula.rb:3223:in `stage'
/opt/homebrew/Library/Homebrew/formula.rb:1582:in `brew'
/opt/homebrew/Library/Homebrew/build.rb:133:in `block in install'
/opt/homebrew/Library/Homebrew/extend/kernel.rb:536:in `with_env'
/opt/homebrew/Library/Homebrew/build.rb:125:in `install'
/opt/homebrew/Library/Homebrew/build.rb:232:in `<main>'
```

```
ameba-1.6.2 % ls -a bin
.               ..              .keep           ameba.cr
```

**Comment by Sija** [comment]
Refs https://github.com/crystal-ameba/ameba/issues/477

**Comment by Sija** [comment]
Should be fixed in v1.9.3
